docker/local-develop.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+function dependency(){
+	(command -v $1 > /dev/null || (echo "install $1" && exit 1))
+}
+
+dependency jq     || exit 1
+dependency docker || exit 2
+dependency grep   || exit 3
+dependency sed    || exit 4
+
+# usage
+echo "execute 'source $0' to get all env vars for testing"
+
+# cleanup
+docker rm -f pg rd 2>/dev/null
+
+# vars
+user=iroha
+password=iroha
```

Comment:
it common practice to name bash vars in CAPITAL case, do you mind to fix?

Reply:
No. :ok_man: